app/src/main/java/org/gdg/frisbee/android/onboarding/FirstStartStep1Fragment.java

```diff
@@ -53,21 +60,39 @@
 
     private static final String ARG_SELECTED_CHAPTER = "selected_chapter";
     @Bind(R.id.chapter_spinner)
-    Spinner mChapterSpinner;
+    AutoCompleteSpinnerView autoCompleteSpinnerView;
     @Bind(R.id.confirm)
-    Button mConfirm;
+    Button mConfirmButton;
     @Bind(R.id.viewSwitcher)
     ViewSwitcher mLoadSwitcher;
-    private ChapterAdapter mSpinnerAdapter;
+    private ChapterAdapter mChapterForCityName;
```

Comment:
Btw, the lists are always in the same ordering.
On 16 Dec 2015 14:21, "Friedger Müffke" <friedger@gmail.com> wrote:

> Having one adapter requires us two handle the filter when showing and
> closing the dialog.
>
> Maybe we could implement an adapter that has a switch whether to be
> filterable or not.
> On 16 Dec 2015 14:17, "Said Tahsin Dane" <notifications@github.com> wrote:
>
>> In
>> app/src/main/java/org/gdg/frisbee/android/onboarding/FirstStartStep1Fragment.java
>> <https://github.com/gdg-x/frisbee/pull/505#discussion_r47773901>:
>>
>> >      @Bind(R.id.viewSwitcher)
>> >      ViewSwitcher mLoadSwitcher;
>> > -    private ChapterAdapter mSpinnerAdapter;
>> > +    private ChapterAdapter mChapterForCityName;
>>
>> Why do they have different layouts? I think having the list in 2 adapters
>> doesn't make sense.
>>
>> What about this? We can make the list always sorted by location and then
>> we can have filter.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/gdg-x/frisbee/pull/505/files#r47773901>.
>>
>